Fix an error that prevents C++ applications from building when including
narray-util.h
https://github.com/tmbarchive/iulib/pull/4
--- colib/narray-util.h.orig
+++ colib/narray-util.h
@@ -31,7 +31,7 @@
 #ifndef h_narray_util__
 #define h_narray_util__
 
-#include <math.h>
+#include <cmath>
 #include <stdlib.h>
 #include "colib/checks.h"
 #include "misc.h"
@@ -176,7 +176,7 @@ namespace colib {
         double total = 0.0;
         for(int i=0;i<a.length1d();i++)
             total += sqr(a.at1d(i)-b.at1d(i));
-        CHECK_ARG(!isnan(total));
+        CHECK_ARG(!std::isnan(total));
         return total;
     }
 
